src/index.js

```diff
@@ -25,144 +25,137 @@ import InstructorsOnly from './components/InstructorsOnly'
+        }
+
+        return (
+          <BrowserRouter>
+            <div>
+
+              <Navigation
+                items={[
+                  {
+                    text: overviewTitleText,
+                    action: '/',
+                  },
+                  {
+                    text: newLessonsTitleText,
+                    action: '/lessons/new',
+                  },
+                  {
+                    text: instructorsTitleText,
+                    action: '/instructors',
+                  },
+                  {
+                    text: guideTitleText,
+                    action: guideUrl,
+                  },
+                  {
+                    text: chatTitleText,
+                    action: chatUrl,
+                  },
+                  {
+                    text: logOutTitleText,
+                    action: logout,
+                  },
+                ]}
               />
 
-              <Route 
-                exact
-                path='/lessons/new'
-                render={() => (
-                  <Request url={`/api/v1/instructors/${user.instructor_id}`}>
-                    {({data}) => (
-                      <New instructor={data} />
+              <Main>
+
+                <Switch>
+
+                  <Route 
+                    exact
+                    path='/'
+                    render={() => (
+                      <Request url={data.instructor_url}>
+                        {({data}) => (
+                          <div>
+                            <Overview instructor={data} />
+                          </div>
+                        )}
+                      </Request>
                     )}
-                  </Request>
-                )}
-              />
-
-              <Route 
-                path={`/lessons/:slug`}
-                render={({match}) => (
-                  <Request url={`/api/v1/lessons/${match.params.slug}`}>
-                    {({data}) => (
-                      <Lesson lesson={data} />
+                  />
+
+                  <Route 
+                    exact
+                    path='/lessons/new'
+                    render={() => (
+                      <Request url={data.instructor_url}>
+                        {({data}) => (
+                          <New instructor={data} />
+                        )}
+                      </Request>
                     )}
-                  </Request>
-                )}
-              />
-
-              <Route 
-                path={`/instructors/:slug`}
-                render={({match}) => (
-                  <Request url={`/api/v1/instructors/${match.params.slug}`}>
-                    {({data}) => (
-                      <Overview instructor={data} />
+                  />
+
+                  <Route 
+                    path={`/lessons/:slug`}
+                    render={({match}) => (
+                      <Request url={`/api/v1/lessons/${match.params.slug}`}>
```

Comment:
this is where `uri_templates` would be nice in the responses. This would be on the user object.